content/100-getting-started/02-setup-prisma/200-add-to-existing-project/110-relational-databases/150-introspection.mdx

```diff
@@ -349,11 +349,11 @@ Run the following command to introspect your database:
 npx prisma db pull
 ```
 
-This commands reads the `DATABASE_URL` environment variable that's defined in `.env` and connects to your database. Once the connection is established, it introspects the database (i.e. it _reads the database schema_). It then translates the database schema from SQL into a Prisma data model.
+This commands reads the `DATABASE_URL` environment variable that's defined in `.env` and connects to your database. Once the connection is established, it introspects the database (i.e. it _reads the database schema_). It then translates the database schema from SQL into a Prisma ORM data model.
```

Comment:
see question above

Reply:
Again this is a product decision. It was my understanding that we wanted all references of Prisma that meant Prisma ORM to be marked as so. In these instances it makes sense that Prisma ORM model and Prisma ORM data model is what we use.

Reply:
To sidestep the problem similar to https://github.com/prisma/docs/pull/5631#discussion_r1490821706:
```suggestion
This commands reads the `DATABASE_URL` environment variable that's defined in `.env` and connects to your database. Once the connection is established, it introspects the database (i.e. it _reads the database schema_). It then translates the database schema from SQL into a data model in your Prisma schema.
```